src/lib/serviceRoutines/postUpdateContext.cpp

```diff
@@ -650,7 +650,64 @@ std::string postUpdateContext
     response.merge(&upcrs);
   }
 
-  TIMED_RENDER(answer = response.render(ciP, UpdateContext, ""));
+  // Note this is a slight break in the separation of concerns among the different layers (i.e.
+  // serviceRoutine/ logic should work in a "NGSIv1 isolated context"). However, it seems to be
+  // a smart way of dealing with partial update situations
+  if (ciP->apiVersion == "v2")
+  {
+    // Adjust OrionError response in the case of partial updates. This may happen in CPr forwarding
+    // scenarios. Note that mongoBackend logic "splits" sucessfull updates and failing updates in
```

Comment:
Fixed in https://github.com/telefonicaid/fiware-orion/commit/3e4cd945bb1ab64ca9dc5507568445f5241b8c32